### PR TITLE
Fixed #29440 -- Document where the `bulk` argument applies in RelatedManager

### PR DESCRIPTION
**Pull request by jlaine**

### CODE REVIEW

File: docs/ref/models/relations.txt [hunk 1]
```diff
@@ -129,7 +129,10 @@ Related objects reference
         :data:`~django.db.models.signals.post_save` signals and comes at the
         expense of performance.
 
-    .. method:: clear()
+        For many-to-many relationships, the ```bulk``` keyword argument does
```
Comment (codingjoe) [inline docs/ref/models/relations.txt]: s/```/``/

File: docs/ref/models/relations.txt [hunk 2]
```diff
@@ -143,6 +146,9 @@ Related objects reference
         :class:`~django.db.models.ForeignKey`\s where ``null=True`` and it also
         accepts the ``bulk`` keyword argument.
 
+        For many-to-many relationships, the ```bulk``` keyword argument does
```
Comment (codingjoe) [inline docs/ref/models/relations.txt]: s/```/``/

File: docs/ref/models/relations.txt [hunk 3]
```diff
@@ -156,7 +162,11 @@ Related objects reference
         If ``clear=True``, the ``clear()`` method is called instead and the
         whole set is added at once.
 
-        The ``bulk`` argument is passed on to :meth:`add`.
+        For :class:`~django.db.models.ForeignKey` objects, the ``bulk``
+        argument is passed on to :meth:`add` and :meth:`remove`.
+
+        For many-to-many relationships, the ```bulk``` keyword argument does
```
Comment (codingjoe) [inline docs/ref/models/relations.txt]: s/```/``/

**Comment by jlaine** [comment]
I've fixed the wrong triple quotes, thanks Johannes

**Comment by jlaine** [comment]
Thanks Tim and Johannes!

**Pull request merged**